### ISSUE TITLE
noise_shape in dropout layers

### ISSUE THREAD

**Issue by YUHAN666** [qubvel/efficientnet#105]
Hi,
Thanks for your fantastic work. May I ask why do you set the `noise_shape` in the `tf.nn.dropout` to (None, 1, 1, 1)?
https://github.com/qubvel/efficientnet/blob/6ceca713c1186168f41a7a628e4d6e2fdc4aa7ef/efficientnet/model.py#L240-L242
Does the **noise_shape=(None, 1, 1, 1)**  means that some data in the batch is randomly abandoned at drop_rate during the training phase?
